garage.com/src/main/java/ConvertibleCar.java

```diff
@@ -0,0 +1,25 @@
+public class ConvertibleCar extends Car implements ICar {
+	boolean TopCanBeLowed;
+	boolean TopCanBeRaised;
+	
+	public ConvertibleCar(String model, IEngine engine, IIgnition ignitionSystem) {
+		super(model, engine, ignitionSystem);
+		
+	}
+	
+	public void top() {
+		this.TopCanBeLowed = true;	
```

Comment:
These field names should describe the Convertible top's state, like "isTopLowered" or "isTopRaised"